ddp-workspace/projects/ddp-rarex/src/app/components/participants-list/participants-list.component.ts

```diff
@@ -85,23 +86,28 @@ export class ParticipantsListComponent implements OnInit {
     });
 
     dialogRef
-      .afterClosed()
-      .pipe(
-        filter((shallDelete: boolean) => shallDelete),
-        tap(() => {
-          this.loading = true;
-        }),
-        mergeMap(() => this.deleteParticipant(userGuid)),
-        tap(() => {
-          this.participants = this.participants.filter(
-            participant => participant.userGuid !== userGuid,
-          );
-        }),
-      )
-      .subscribe(() => {
-        this.loading = false;
-        this.expandedMap[userGuid] = null;
-      });
+        .afterClosed()
+        .pipe(
+            filter((shallDelete: boolean) => shallDelete),
+            tap(() => {
+              this.loading = true;
+              this.errorMessage = null;
+            }),
+            mergeMap(() => this.deleteParticipant(userGuid)),
+            tap(() => {
+              this.participants = this.participants.filter(
+                  participant => participant.userGuid !== userGuid,
+              );
+            }),
+            catchError(() => {
+              this.errorMessage = this.translateService.instant('ParticipantsList.DeleteError');
```

Comment:
This is where the errorMessage gets set!
Note that when catchError executes `subscribe` still gets executed too, but the `tap` above that updates `participants` does not.